AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -105,14 +147,30 @@ func addDoccDependency() {
 
 // MARK: - Services
 
+let serviceTargetDependencies: [Target.Dependency] = [
```

Comment:
This long list is broken into a variable so it can be shared between service client targets and protocol test targets.